packages/terra-popup/tests/nightwatch/components/BoundedHeightPopup.jsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import Popup from '../../../lib/Popup';
+
+class BoundedPopup extends React.Component {
```

Comment:
![popup-bounded](https://user-images.githubusercontent.com/8146159/27056698-d5d8cfe2-4f8e-11e7-89a3-b7143e0773a9.gif)
The bounded popup will move when tabbing to an element outside the viewable portion of the page.